api/sync.js

```
@@ -23,13 +23,15 @@ router.use(express.json({ limit: '1mb' }));
 const _store = new Map(); // userId → { presets, noiseProfiles, history, updatedAt }
 
 // ─── Auth Middleware ──────────────────────────────────────────────────────────
+// FIX: no throw on missing env var so Vercel deployments without the secret
+// don't crash at startup. Set LICENSE_JWT_SECRET in Vercel Environment Variables.
 const LICENSE_SECRET = (() => {
   if (process.env.LICENSE_JWT_SECRET) return process.env.LICENSE_JWT_SECRET;
```

ESLint

[error] 'process' is not defined.

(no-undef)